src/javascript/_common/base/client_base.js

```diff
@@ -403,6 +403,13 @@ const ClientBase = (() => {
         return options_blocked_countries.includes(country);
     };
 
+    const isMalta = () => {
+        const malta_countries = ['at', 'be', 'bg', 'cy', 'cz', 'dk', 'ee', 'fi', 'hr', 'hu', 'ie', 'lt', 'lv', 'nl', 'pl', 'pt', 'ro', 'se', 'si', 'sk'];
```

Comment:
Would it be possible to use `landing_company_shortcode` to check if it's `malta` instead?
I think you can access it like that `const current_landing_company = get('landing_company_shortcode');`. Or perhaps we can check it directly in `markets.jsx` if it's accessible there.

Reply:
the landing_company_shortcode is 'virtual' when we using virtual account in any landing_company and we can not understand it is the virtual account of Malta client

Reply:
Oh yeah, this makes sense. We'll leave it like that then 👍🏻